src/routes/v2/authenticatedSites/settings.js

```diff
@@ -136,7 +135,7 @@ class SettingsRouter {
     router.post(
       "/repo-password",
       this.authorizationMiddleware.verifyIsEmailUser,
-      attachWriteRouteHandlerWrapper(this.updateRepoPassword)
+      attachReadRouteHandlerWrapper(this.updateRepoPassword)
```

Comment:
shouldn't updating a repo's password be a write op?

Reply:
[79cdf6a](https://github.com/isomerpages/isomercms-backend/pull/1044/commits/79cdf6a57e0888d1011884f852ef834e562605ca)